buyer-api/src/routes/dataOrders/offchainData.js

```diff
@@ -42,6 +43,11 @@ router.get(
       closedAt,
       ...offchainData
     } = req.dataOrder;
+    offchainData.notaries = await Promise.all(offchainData.notariesAddresses
+      .map(async (notaryAddress) => {
+        const notary = await notaries.fetch(notaryAddress);
+        return notary;
+      }));
```

Comment:
Since the `Promise.all` is going to take care of the map's resulting promises, I'd replace this map with:
```js
      .map((notaryAddress) => notaries.fetch(notaryAddress)));
```